rubin_scheduler/scheduler/model_observatory/kinem_model.py

```diff
@@ -435,34 +520,58 @@ def slew_times(
                     self.current_ra_rad, self.current_dec_rad, mjd
                 )
 
+        # Now calculate how far we need to move,
+        # in altitude, azimuth, and camera rotator (if applicable).
+        # Delta Altitude
         delta_alt = np.abs(alt_rad - starting_alt_rad)
+        # Delta Azimuth - there are two different 'directions'
```

Comment:
Reduce use of "scare quotes" in comments and doc strings. The "directions" have well-understood names of "clockwise" and "anti-clockwise". A scare quote implies you are using a euphemism, and folks reading comments usually want explicit descriptions and not euphemisms.